cedar-policy/CHANGELOG.md

```diff
@@ -97,6 +93,22 @@ Cedar Language Version: 4.0
 - (*) `Entities::from_entities()` will now correctly reject record
   attributes with superfluous attributes. (#1177, resolving #1176)
 
+## [3.4.0] - Coming soon
+Cedar Language Version: 3.4
```

Comment:
Question for review: do we want to bump the language version due to #1177? My inclination is not to, since this was a bug fix.

Reply:
Seconded